applications/openshift/worker/file_permissions_kubelet_conf/rule.yml

```diff
@@ -5,7 +5,7 @@ prodtype: ocp4
 title: 'Verify Permissions on The Kubelet Configuration File'
 
 description: |-
-    {{{ describe_file_permissions(file="/etc/kubernetes/kubelet.conf", perms="0600") }}}
+    {{{ describe_file_permissions(file="/etc/kubernetes/kubelet.conf", perms="0644") }}}
```

Comment:
This is a good time to ask: why permissions are moving from more secure to insecure? This is kinda backwards, and if we are doing default file permissions also makes me think that there is no value in file permission checks now. This is a question as to why this is being done.

Reply:
The check was probably written to check for 0600, only to now discover that the default is actually 644. We might want to start restricting these, as was done for the audit logs https://github.com/openshift/cluster-openshift-apiserver-operator/pull/397

Reply:
+1 to tracking hardening the permissions in a separate ticket. Currently the changes to the rule reflect both the CIS benchmark and the permissions on the cluster.

Reply:
If we can open a ticket for all permissions in CIS benchmark to be `0600` instead of `0644` and add the ticket here, that would be great.